src/views/Drive.vue

```diff
@@ -1109,12 +1112,11 @@ module.exports = {
                         //resetting .value tricks browser into allowing subsequent upload of same file(s)
                         document.getElementById('uploadFileInput').value = "";
                         document.getElementById('uploadDirectoriesInput').value = "";
-                        let future = peergos.shared.util.Futures.incomplete();
                         let progressBars = [];
                         for(var i=0; i < files.length; i++) {
                             let that = this;
-                            var thumbnailAllocation = Math.min(100000, files[i].size / 10);
-                            var resultingSize = files[i].size + thumbnailAllocation;
+                            //var thumbnailAllocation = Math.min(100000, files[i].size / 10);
```

Comment:
is the thumbnailAllocation still meant to be commented?

Reply:
The thumbnailAllocation code was a bit of a hack. Previously we would fill the progress bar to 100% when the upload had finished (post thumbnail creation). No longer get a notification when individual upload finished. I have now removed the code.